src/libs/Navigation/types.ts

```diff
@@ -388,7 +393,10 @@ type SettingsNavigatorParamList = {
     [SCREENS.GET_ASSISTANCE]: {
         backTo: Routes;
     };
-    [SCREENS.SETTINGS.TWO_FACTOR_AUTH]: BackToParams;
+    [SCREENS.SETTINGS.TWO_FACTOR_AUTH]: {
```

Comment:
Can we use `BackToAndForwardToParms` here and in place of `SuccessStepParams` in `SuccessStep`?

Also, in which cases is `forwardTo` applicable?

Reply:
i don't see any other reuse, so this is fine as is